ext/comprehensions_test.go

```diff
@@ -345,6 +345,62 @@ func TestTwoVarComprehensionsRuntimeErrors(t *testing.T) {
 	}
 }
 
+func TestTwoVarComprehensionsUnparse(t *testing.T) {
```

Comment:
could you update this test to include a PruneAST example that would have failed?

Reply:
I think all of these tests would have failed since the second variable would have been omitted. Happy to add more cases if you have some in mind

Reply:
Oh, of course, I should actually add tests for pruning. Sorry about that